ts/src/pro/bitcoincom.ts

```diff
@@ -2,17 +2,16 @@
 //  ---------------------------------------------------------------------------
 
 import hitbtc from './hitbtc.js';
-import bequantRest from '../bequant.js';
+import hitbtcRest from '../hitbtc.js';
```

Comment:
idk why i had bequant there. even though bequant derives from hitbtc and result would have been same, i had made a mistake